frontend/its-our-call/src/components/SearchOverlay.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+
+interface SearchOverlayProps {
+  hidden: boolean;
+}
+
+interface SearchOverlayState { }
+
+class SearchOverlay extends React.Component<SearchOverlayProps, SearchOverlayState> {
+  render() {
```

Comment:
I would add an `if this.props.hidden return null` or else the `hidden` prop means nothing from within the component itself.

Reply:
Could you explain this a little further? I think I see how using `if this.props.hidden return null` is a more flexible pattern, but would assigning it to the `hidden` property of the underlying div not have the same effect?